compiler/rustc_codegen_llvm/src/back/write.rs

```diff
@@ -548,6 +548,11 @@ pub(crate) unsafe fn optimize(
                     llvm::LLVMRustAddPass(fpm, find_pass("lint").unwrap());
                     continue;
                 }
+                if pass_name == "insert-gcov-profiling" || pass_name == "instrprof" {
+                    // Instrumentation should be inserted before optimization.
```

Comment:
Can we say "must" instead of "should" and reference the Issue number?

Reply:
I couldn't find a suitable issue to point to so I just pointed at the Clang code.

Reply:
You're updated comment works for me. Thanks!